### PR TITLE
feat(email): Added support for agent status in email

### PR DESCRIPTION
**Pull request by GMishx**
**Prerequisite:**

1.  Install heirloom-mailx (BSD mail client, formally known as "nail").

    `sudo apt-get install heirloom-mailx`

**Enhancement:**

1.  New AGENTSTATUS macro for list of passed or failed agents.
2.  Complete path of the folder where the package is uploaded.

pls see also: https://github.com/fossology/fossology/wiki/Email-notification-configuration

### CODE REVIEW

**Comment by mcjaeger** [comment]
It seem s to be a difficult topic and I think some documentation for admin is required (maybe in the wiki of this project)

**Comment by ag4ums** [comment]
replace sampleheader and samplefooter during installation.

**Comment by GMishx** [comment]
I have moved the configurations required for heirloom-mailx to the sysconfig table in the database which can be modified in the page Customize under the Admin tab.
The wiki page is updated accordingly.

**Comment by ag4ums** [comment]
fo-postinstall in failing on Indexes, please rebase with master

**Comment by GMishx** [comment]
Rebased with master.

**Comment by ag4ums** [comment]
fo-postinstall is failing for this branch
image

**Comment by GMishx** [comment]
I have resolved it and pushed the new commit again.

File: src/scheduler/agent/Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ######################################################################
 # Copyright (C) 2010-2012 Hewlett-Packard Development Company, L.P.
-# Copyright (C) 2015 Siemens AG
+# Copyright (C) 2018 Siemens AG
```
Comment (mcjaeger) [inline src/scheduler/agent/Makefile]: pls do not forget to make it 2015,2018

File: src/scheduler/agent/database.c [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /* **************************************************************
 Copyright (C) 2010, 2011, 2012 Hewlett-Packard Development Company, L.P.
-Copyright (C) 2015 Siemens AG
+Copyright (C) 2018 Siemens AG
```
Comment (mcjaeger) [inline src/scheduler/agent/database.c]: again...

File: src/scheduler/agent/sqlstatements.h [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /* **************************************************************
 Copyright (C) 2012 Hewlett-Packard Development Company, L.P.
-Copyright (C) 2015 Siemens AG
+Copyright (C) 2018 Siemens AG
```
Comment (mcjaeger) [inline src/scheduler/agent/sqlstatements.h]: copyright

**Comment by GMishx** [comment]
Applied the changes suggested by @mcjaeger

**Pull request merged**